rbac/management/migrations/0012_remove_RonR_resources.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.4 on 2019-11-26 17:03
+
+from django.db import migrations, models
+
+
+def remove_rbac_roles(apps, schema_editor):
+    # get all permissions/access objects for RBAC
+    Access = apps.get_model('management', 'Access')
+    rbac_permissions = Access.objects.filter(permission__contains='rbac:')
+
+    # iterate through all RBAC permissions
+    for rbac_permission in rbac_permissions:
+        # get the role for the permission (bubbling up)
+        role = rbac_permission.role
+
+        # check to see if there are any access objects that are not RBAC
+        non_rbac_permissions = role.access.exclude(permission__contains='rbac:')
+
+        # if so, just delete the access object we know is RBAC, and leave the role
+        if non_rbac_permissions:
+            # this will still delete the resource definitions for the access object
+            access.delete()
```

Comment:
This actually needs to be `rbac_permission` here, I believe?
```suggestion
            rbac_permission.delete()
```

Reply:
Yep, I think you're right. That's what I get for playing with naming just before submitting >_<